VirtoCommerce.Domain/Search/IndexDocumentSource.cs

```diff
@@ -1,8 +1,20 @@
-﻿namespace VirtoCommerce.Domain.Search
+using VirtoCommerce.Domain.Search.ChangeFeed;
+
+namespace VirtoCommerce.Domain.Search
 {
     public class IndexDocumentSource
     {
         public IIndexDocumentBuilder DocumentBuilder { get; set; }
+
```

Comment:
You might mark it as [Obsolete] with IIndexDocumentChangesProvider type.  (optional)